companies/contentfleet.json

```
@@ -0,0 +1,20 @@
+{
+    "slug": "contentfleet",
+    "relevant-countries": [
+        "de"
+    ],
+    "name": "Content Fleet GmbH",
+    "runs": [
+        "einfachtierisch.de"
+    ],
+    "address": "Kehrwieder 8\n20457 Hamburg",
+    "email": "datenschutz@contentfleet.com",
```

```suggestion
    "email": "datenschutz@contentfleet.com",
    "phone": "+49 30 12085507",
    "fax": "+49 40 600800619",
```
added phone and generic fax

---

Not exactly important but I prefer to keep them in the same order as in the schema.

```suggestion
    "phone": "+49 30 12085507",
    "fax": "+49 40 600800619",
    "email": "datenschutz@contentfleet.com",
```